testing/scripts/playwright_test.py

```
@@ -2935,6 +2935,145 @@ async def test_ipv6_mapped6_ui(page: Page) -> None:
+              data6.get("data", {}).get("ipv4"), "192.0.2.1")
+
+    # No embedding
+    status7, data7 = _api_post("embedded-v4", {"input": "2001:db8::1"})
+    assert_eq("api embedded-v4 (none): HTTP 200", status7, 200)
+    assert_eq("api embedded-v4 (none): scheme is null",
+              data7.get("data", {}).get("scheme"), None)
+    assert_eq("api embedded-v4 (none): ipv4 is null",
+              data7.get("data", {}).get("ipv4"), None)
+
+    # Loopback / unspecified must NOT be misdetected as compatible (RFC 4291 §2.5.5.1)
+    status8, data8 = _api_post("embedded-v4", {"input": "::1"})
```

Ruff (0.15.12)

[warning] Unpacked variable `status8` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)